applications/mp/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>io.helidon.applications</groupId>
         <artifactId>helidon-applications-project</artifactId>
-        <version>1.4.1-SNAPSHOT</version>
+        <version>1.4.1</version>
```

Comment:
Danger! Danger!

Reply:
Huh, not sure what happened here. Will merge.

Reply:
Done.

Reply:
I don't like having that merge here, going to redo since it is such a simple change.